src/main.cpp

```diff
@@ -157,27 +161,47 @@ void wrongDataState(){
  * Average methods
  **/
 
+#ifndef SENSIRION
 void saveDataForAverage(unsigned int pm25, unsigned int pm10){
   v25.push_back(pm25);
   v10.push_back(pm10);
 }
+#else
+void saveDataForAverage(unsigned int pm25, unsigned int pm10, float pm25f){
+  v25.push_back(pm25);
+  v10.push_back(pm10);
+  v25f.push_back(pm25f);
```

Comment:
this vector, v25f, is a float vector, you tested it with influxdb old schema? or it don't matter?

Reply:
This is a float vector that carries the float value of PM2.5 of the Sensirion sensor, is sending to the influx and gblabs server via the speed variable in the frame that is float too. I think is provisional for the test that I'm making

Reply:
For a test code, you need write a independent code for run test units or similar kind tests code.

Reply:
deleted the pm25f